pkg/stacker/build.go

```diff
@@ -476,7 +476,7 @@ func (b *Builder) build(s types.Storage, file string) error {
 			}
 
 			// These should all be non-interactive; let's ensure that.
-			err = c.Execute("/stacker/.stacker-run.sh", nil)
+			err = c.Execute("/stacker/imports/.stacker-run.sh", nil)
```

Comment:
this feels weird in /stacker/imports, but it seems unlikely to stomp on user content. so unless you think of something that fits better, this seems fine.

Reply:
Maybe we can put this under /stacker/tools/.stacker-run.sh

Reply:
Actually, this is under "automatic" binding of /stacker/imports, let's not change that.